.rat-excludes

```diff
@@ -15,15 +15,15 @@ TAGS
 RELEASE
 control
 docs
-docker.properties.template
-fairscheduler.xml.template
-spark-defaults.conf.template
+docker.properties.*
+fairscheduler.xml.*
+spark-defaults.conf.*
 log4j.properties
-log4j.properties.template
+log4j.properties.*
 metrics.properties
-metrics.properties.template
+metrics.properties.*
 slaves
-slaves.template
+slaves.*
```

Comment:
this is casting a net too wide -- i.e. one day we might have slaves.scala and this won't catch it.